tests/base/requirements-w-hashes.txt

```diff
@@ -87,3 +87,5 @@ werkzeug==0.14.1 \
     --hash=sha256:c3fd7a7d41976d9f44db327260e263132466836cef6f91512889ed60ad26557c \
     --hash=sha256:d5da73735293558eb1651ee2fddc4d0dedcfa06538b8813a2e20011583c9e49b \
     # via flask
+futures==3.2.0 \
+    --hash=sha256:ec0a6cb848cc212002b9828c3e34c675e0c9ff6741dc445cab6fdd4e1085d1f1 \
```

Comment:
Why was this added? I looks incorrect to me. I think removing it may allow the tests to pass.